exercises/concept/black-jack/.meta/exemplar.py

```diff
@@ -1,23 +1,72 @@
-def number_of_card(card):
-    if card == 'A':
-        return "ace"
-    elif card == 'J' or card == 'Q' or card == 'K':
-        return 10
+def value_of_card(card):
+    """
+
+    :param card: str - given card.
+    :return: int - value of a given card (face card = 10, pip value otherwise).
+    """
+
+    if card == 'J' or card == 'Q' or card == 'K':
+        value = 10
     else:
-        return card
+        value = int(card)
+    return value
+
+
+def value_of_ace(hand_value):
+    """
 
+    :param hand_value: int - current hand value.
+    :return: int - value of the upcoming ace card (either 1 or 11).
+    """
 
-def number_of_ace(hand):
-    if hand + 11 <= 21:
-        return 11
+    if hand_value + 11 > 21:
+        value = 1
     else:
-        return 1
+        value = 11
+    return value
 
 
-def blackjack(hand):
-    if 'A' not in hand:
-        return False
-    elif 'J' in hand or 'Q' in hand or 'K' in hand or 10 in hand:
-        return True
+def is_blackjack(card_one, card_two):
+    """
+
+    :param card_one: str - first card in hand.
+    :param card_two: str - second card in hand.
+    :return: bool - if the hand is a blackjack (two cards worth 21).
+    """
+
+    if card_one == 'A' and card_two != 'A':
+        blackjack = value_of_card(card_two) == 10
+    elif card_one != 'A' and card_two == 'A':
+        blackjack = value_of_card(card_one) == 10
     else:
-        False
+        blackjack = False
+    return blackjack
+
+
+def can_split_pairs(card_one, card_two):
+    """
+
+    :param card_one: str - first card in hand.
+    :param card_two: str - second card in hand.
+    :return: bool - if the hand can be split into two pairs (i.e. cards are of the same value).
+    """
+
+    if card_one == 'A' or card_two == 'A':
+        split_pairs = card_one == card_two
+    else:
+        split_pairs = value_of_card(card_one) == value_of_card(card_two)
+    return split_pairs
+
+
+def can_double_down(card_one, card_two):
+    """
+
+    :param card_one: str - first card in hand.
+    :param card_two: str - second card in hand.
+    :return: bool - if the hand can be doubled down (i.e. totals 9, 10 or 11 points).
+    """
+
+    card_one_value = 1 if card_one == 'A' else value_of_card(card_one)
+    card_two_value = 1 if card_two == 'A' else value_of_card(card_two)
```

Comment:
Couldn't think of a better way to solve this without iterating over something. It's just two lines so maybe not that bad.

Reply:
I think that's fine.  We may want to describe either in `hints.md` or in the `introduction.md` how to make a `ternary` in Python.  

Something along the lines of  (_but better phrasing_) "when `if` | `else` statements are short, they can be combined in one line [example].  This is what is known in other programming languages as a `ternary expression`, and usually includes a special operator.  Python does not have a special operator for its `ternary expression`.

Reply:
I chose the `hints.md` route.